library/styleguideComponents/Wrapper/index.js

```diff
@@ -77,54 +89,57 @@ export default function Wrapper(props) {
 
     return (
         <Application theme={theme}>
-            <StyledTopBar>
-                <StyledLeftElement>
-                    <ButtonMenu buttonVariant="base" icon={<InfoFilled />}>
-                        <StyledInfoContent>
-                            <StyledContentText>
-                                The theme support is
-                                <StyledBadge label="BETA" />
-                                and
-                                <StyledBadge label="WIP" />
-                            </StyledContentText>
-                        </StyledInfoContent>
-                    </ButtonMenu>
-                    <StyledTitle>Info</StyledTitle>
-                </StyledLeftElement>
-                <StyledPickerTheme
-                    menuAlignment="right"
-                    menuSize="x-small"
-                    icon={<StyledLogo src={logo} alt="rainbow logo" />}
-                >
-                    <MenuItem label="THEME SELECTOR" variant="header" />
-                    <MenuItem
-                        label="Default"
-                        icon={<StyledColorBox />}
-                        onClick={() => setTheme()}
-                    />
-                    <MenuItem
-                        label="Orange"
-                        icon={<StyledColorBox rainbowTheme={orangeTheme.rainbow} />}
-                        onClick={() => setTheme(orangeTheme)}
-                    />
-                    <MenuItem
-                        label="Cyan"
-                        icon={<StyledColorBox rainbowTheme={cyanTheme.rainbow} />}
-                        onClick={() => setTheme(cyanTheme)}
-                    />
-                    <MenuItem
-                        label="Yellow"
-                        icon={<StyledColorBox rainbowTheme={yellowTheme.rainbow} />}
-                        onClick={() => setTheme(yellowTheme)}
-                    />
-                    <MenuItem
-                        label="Deep Purple"
-                        icon={<StyledColorBox rainbowTheme={deepPurpleTheme.rainbow} />}
-                        onClick={() => setTheme(deepPurpleTheme)}
-                    />
-                </StyledPickerTheme>
-            </StyledTopBar>
-            <Provider store={store}>{children}</Provider>
+            <StyledWrapper>
+                <StyledTopBar>
+                    <StyledLeftElement>
+                        <InfoFilled />
+                        <StyledTitle>
+                            The theme support is <StyledBadge label="BETA" /> and{' '}
+                            <StyledBadge label="WIP" />
+                        </StyledTitle>
+                    </StyledLeftElement>
+                    <StyledPickerTheme
```

Comment:
We need to add the checkmark for the option selected on the button menu
<img width="780" alt="Screen Shot 2019-12-17 at 9 28 54 AM" src="https://user-images.githubusercontent.com/32083556/71009371-b833f580-20af-11ea-8e4b-2a0fd1698745.png">